recipes/django-param/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "django_param" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/django-param/{{ name }}-{{ version }}.tar.gz
+  sha256: 8979f21a2a805ed3e87af65428dac15a09ffb5c4b6567d5f9b082751738a48ae
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - setuptools
```

Comment:
Is setuptools needed at runtime?

Reply:
probably not. I will remove it. I think it was an artifact from reusing a meta file

Reply:
removed